docs/sources/update_docs.sh

```diff
@@ -6,8 +6,8 @@ cp -r build/html ../html
 make clean
 
 # generate mandarin doc
-echo "begin to generate mandarin html"
-cd ../docs_zh/sources
-./update_docs.sh
-cp -r ../zh ../../html/
-echo "Finish"
+#echo "begin to generate mandarin html"
```

Comment:
why did you comment all that out?

Reply:
because ZH version won't be generated that way...

https://docs.readthedocs.io/en/stable/guides/manage-translations.html

Reply:
so right after this merge, how ZH docs will be generated? I don't understand what this script has to do with RTD?